addons/hearing/initSettings.inc.sqf

```diff
@@ -11,6 +11,15 @@ private _category = format ["ACE %1", LLSTRING(Module_DisplayName)];
     true // Needs mission restart
 ] call CBA_fnc_addSetting;
 
+[
+    QGVAR(explosionDeafnessCoefficient),
+    "SLIDER",
+    [LSTRING(explosionDeafnessCoefficient_DisplayName), LSTRING(explosionDeafnessCoefficient_Description)],
+    _category,
+    [0, 5, 1, 2],
+    1
```

Comment:
Unsure if this should be enforced or left to individuals.

Reply:
I want 0 to be an option, so that people can revert to prior behaviour - leaving it at enforced then.